packages/react-components/react-toolbar/src/stories/Toolbar/ToolbarVerticalButton.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { FontIncrease24Regular, FontDecrease24Regular, TextFont24Regular } from '@fluentui/react-icons';
+import { Toolbar, ToolbarButton } from '@fluentui/react-toolbar';
+import type { ToolbarProps } from '@fluentui/react-toolbar';
+
+export const VerticalButton = (props: Partial<ToolbarProps>) => (
+  <Toolbar {...props}>
+    <ToolbarButton vertical appearance="primary" icon={<FontIncrease24Regular />}>
```

Comment:
we shouldn't use sized icons here. What should happen rather is that when the `vertical` prop is used, we should apply a `fontSize` style in css to size an unsized icon

Reply:
Done

Reply:
What I meant is that the slot should be styled using `fontSize` in **css**.

```ts
const useStyles = makeStyles({
  verticalIcon: {
    fontSize: '24px',
  }
})

const styles = useStyles();
mergeClasses(state.vertical && verticalIcon);
```

Reply:
Hmm, help me understand why? We would prevent users to set whatever icon size they want?